pkg/controller/pod_volume_backup_controller.go

```diff
@@ -258,12 +258,12 @@ func (r *PodVolumeBackupReconciler) getParentSnapshot(ctx context.Context, log l
 			continue
 		}
 
-		if mostRecentPVB == nil || pvb.Status.StartTimestamp.After(mostRecentPVB.Status.StartTimestamp.Time) {
-			mostRecentPVB = &pvb
+		if mostRecentPVB.Status == (velerov1api.PodVolumeBackupStatus{}) || pvb.Status.StartTimestamp.After(mostRecentPVB.Status.StartTimestamp.Time) {
+			mostRecentPVB = pvb
 		}
 	}
 
-	if mostRecentPVB == nil {
+	if mostRecentPVB.Status == (velerov1api.PodVolumeBackupStatus{}) {
```

Comment:
The current change has one precondition/assumption: for a normal PVB, its .Status never equals to velerov1api.PodVolumeBackupStatus{}. Otherwise, if the PVB, which should have been the search target, indeed has a Status == velerov1api.PodVolumeBackupStatus{}, this judgement will lead to a fault result.

Reply:
@Lyndon-Li the Line # 246 `if pvb.Status.Phase != velerov1api.PodVolumeBackupPhaseCompleted `  which will filter the empty pvb.Status case， so once `mostRecentPVB` has been assigned value, its Status should not empty. If its Status is empty which means No completed PodVolumeBackup found for PVC